ui/src/pages/backoffice/Item/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -14,12 +15,16 @@ export default class ItemMetadata extends Component {
   render() {
     const { itemDetails } = this.props;
 
-    const metadata = [
+    let metadata = [
```

Comment:
For the `value` properties I would suggest to use `_get` in case the prop is missing from metadata, i.e. 
`value: _get(itemDetails, 'metadata.acquisition_pid')` and provide a sensible default if needed.